guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
+      </para>
+      
+      <para><application>&app;</application> can automatically create lots or a
+        user may create lots. <application>&app;</application> can automatically
+        link buy and sell transaction splits to lots or a user may do the 
+        linking.
+      </para>
+
+      <para>
+        <application>&app;</application> can automatically <emphasis>scrub
+        </emphasis> a security account. <application>&app;</application>
+        scrubbing, when referring to investments, involves creating lots
+        linking sell transaction splits to buy transaction splits, and then 
+        creating a gain/loss transaction for the difference between the sale
+        value and the cost of the securities sold. This can be a useful feature
+        reducing manual calculation especially if dividends have been
+        reinvested over years and there are many different costs involved
+        in a sale.
+      </para>
+
+      <para>The term <emphasis>scrub</emphasis> is used because security
+        accounts need to be cleaned up after sales to ensure the 
+        difference between the cost paid for securities, and value
+        received from selling them, is accounted for as capital gain or
+        loss. If the capital gain/loss is not correct, the Trial Balance
+         (<menuchoice>
+            <guimenu>Reports</guimenu>
+            <guisubmenu>Income &amp; Expense</guisubmenu>
+            <guimenuitem>Trial Balance</guimenuitem>
+          </menuchoice>) bottom line total debits will not balance to 
+          total credits.
+      </para>
+                    
+      <sect3 id="invest-sellLimitations">
+        <title>Limitations</title>
+        
+        <para>There are some limitations to using this feature.</para>
+        
+        <itemizedlist>
+          <listitem>
+            <para><application>&app;</application> has only 1 method of
+              automatically linking buy transactions to sell transactions -
+              First In/ First Out (FIFO). I.e. The oldest securities are always
+              sold first.
+            </para>
+          </listitem>
+          
+          <listitem>
+            <para>The Advanced Portfolio Report does not use lot information
+              when calculating costs. It calculates the cost basis and gains
+              or losses based on the selected <emphasis>Basis calculation method
+              </emphasis> report option (Average, FIFO or LIFO). I.e. it does
+              not use any of the lot information, just the security transaction
+              splits.
+            </para>
+          
+            <para>If you are selling <emphasis>all</emphasis> of your holding of
+              a particular security, it doesn't matter which cost method you
+              use as the total cost of the securities sold will be the same.
+              It is only when you have partially sold a security, that the 
```

Comment:
Tense change, from present ("If you are selling") to past-perfect ("you have partially sold").